jenkins/jenkins-install.sh

```diff
@@ -77,6 +91,7 @@ else
     sudo apt install -yq wget
 fi
 
+echo "Checking for the jenkins command line interface library..."
```

Comment:
Where is the issue tracking that we need to do this work? We need to prevent "busy work" where we're just adding features that are not in scope of what the original project ask is. How could we better propose features?